Notebooks/Exploration.ipynb

```diff
@@ -64,7 +67,7 @@
     "    for state in STATES:\n",
     "        df.loc[df['Province/State'] == state, \"Deaths\"] = df[df['Province/State'] == state].Deaths.transform(lambda x: x*(1/x.max())).fillna(0)\n",
     "        df.loc[df['Province/State'] == state, \"Infected\"] = df[df['Province/State'] == state].Infected.transform(lambda x: x*(1/x.max())).fillna(0)\n",
-    "        df.loc[df['Province/State'] == state, \"Recovered\"] = df[df['Province/State'] == state].Recovered.transform(lambda x: x*(1/x.max())).fillna(0)\n",
```

Comment:
Recovered is not included in retrieval function. This needs to be mapped to be considered in other notebooks